scripts/tasks/eslint.js

```diff
@@ -10,16 +10,22 @@
 const minimist = require('minimist');
 const runESLint = require('../eslint');
 
-console.log('Linting all files...');
-// https://circleci.com/docs/2.0/env-vars/#circleci-environment-variable-descriptions
-if (!process.env.CI) {
-  console.log('Hint: run `yarn linc` to only lint changed files.');
-}
+async function main() {
+  console.log('Linting all files...');
+  // https://circleci.com/docs/2.0/env-vars/#circleci-environment-variable-descriptions
+  if (!process.env.CI) {
+    console.log('Hint: run `yarn linc` to only lint changed files.');
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  const {_, ...cliOptions} = minimist(process.argv.slice(2));
```

Comment:
I had to add destructuring to prevent passing `_`, which `minimist` generates by default. Otherwise, `ESlint` returns this error:

<details>

```
yarn linc
yarn run v1.22.17
$ node ./scripts/tasks/linc.js
Linting changed files...
Lint passed for changed files.
(node:84712) UnhandledPromiseRejectionWarning: Error: Invalid Options:
- Unknown options: _
    at processOptions (/Users/XXX/projects/react/node_modules/eslint/lib/eslint/eslint.js:273:15)
    at new ESLint (/Users/XXX/projects/react/node_modules/eslint/lib/eslint/eslint.js:416:34)
```

</details>